playwright/device-image-size.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect, Locator, Page } from '@playwright/test';
+
+interface ImageSize {
+  width: number;
+  height: number;
+}
+type NullableImgSize = ImageSize | null;
+
+async function setImageAndViewport(
```

Comment:
looks like utils function, i think will be better if we put this function in the folder scope for this test in utils.ts file for specific test